.github/workflows/pull_request.yml

```diff
@@ -136,7 +136,7 @@ jobs:
 
   tv_and_watchos:
     runs-on: macos-latest
-    timeout-minutes: 30
+    timeout-minutes: 40
```

Comment:
F

Reply:
F

Reply:
Seriously though, not sure why it sometimes flukes. It runs on avarage in about 20 minutes, but I guess ocassionally it runs on a machine with less resources?